jazzy/distribution.yaml

```
@@ -0,0 +1,14 @@
+%YAML 1.1
+# ROS distribution file
+# see REP 143: http://ros.org/reps/rep-0143.html
+---
+release_platforms:
+  debian:
+  - bookworm
+  rhel:
+  - '9'
+  ubuntu:
+  - noble
+repositories: []
```

```suggestion
repositories: {}
```
Repositories is associative / dictish / mapish not an array.